src/plugins/node-workspace.ts

```diff
@@ -343,6 +343,22 @@ export class NodeWorkspace extends WorkspacePlugin<Package> {
   protected pathFromPackage(pkg: Package): string {
     return pkg.location;
   }
+
+  private detectRangePrefix(version: string): string {
```

Comment:
non-blocking: we could consider defining an enum of possible prefix types. This could also be done in a cleanup at a later time as these are private interfaces.

Reply:
Added an enum!